governance/polls/Approve SPF ETHAmsterdam Event (MIP55c3-SP3) - March 28, 2022.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Approve SPF ETHAmsterdam Event (MIP55c3-SP3) - March 28, 2022
+summary: Signal your support or opposition to approving SPF ETHAmsterdam Event (MIP55c3-SP3)
+discussion_link: https://forum.makerdao.com/t/mip55c3-sp3-ethamsterdam-event-spf/13781
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-03-28T16:00:00
+end_date: 2022-03-31T16:00:00
+---
+# Poll: Approve SPF ETHAmsterdam Event (MIP55c3-SP3) - March 28, 2022
+
+The Governance Facilitators have placed this poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP4](https://mips.makerdao.com/mips/details/MIP4). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, March, 28 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to approve the [Special Purpose Fund for ETHAmsterdam Event - MIP55c3-SP3](https://github.com/makerdao/mips/blob/master/MIP55/MIP55c3-Subproposals/MIP55c3-SP3.md).
+
+The proposal author offered the following [motivation for this SPF request](https://github.com/makerdao/mips/blob/master/MIP55/MIP55c3-Subproposals/MIP55c3-SP3.md#motivation):
+
+*Recently, MakerDAO hosted an event at the ETHDenver conference called DAIvinity. Due to its overwhelming success and response from the Maker Community we believe that it would be good to host another event during ETHAmsterdam.*
+
+
+Please review the discussion [thread](https://forum.makerdao.com/t/mip55c3-sp3-ethamsterdam-event-spf/13781) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* A one-time transfer of 50,000 DAI from the Maker Protocol to a TBD multi-sig address whose comptrollers are @juan, @hexonaut, and @nadia following the conclusion of this poll.
+
+**If the votes for the 'Yes' option do not exceed the 'No' option then no further action will be taken at this time.**
+
+## Resources
+
+[MIP55](https://mips.makerdao.com/mips/details/MIP55) describes how the Special Purpose Fund works.
```

Comment:
Maybe stretch this out some. "...describes the Special Purpose Fund structure and application process." Or something.